packages/hardhat-ethers/test/index.ts

```diff
@@ -966,15 +942,8 @@ describe("Ethers plugin", function () {
             { args: ["Hello"] }
           );
 
-          assert.containsAllKeys(contract.interface.functions, [
-            "setGreeting(string)",
-            "greet()",
-          ]);
-
-          assert.equal(
-            await contract.signer.getAddress(),
-            await signers[0].getAddress()
-          );
+          assert(await contract.greet(), "Hello");
+          await assertContract(contract, signers[0]);
```

Comment:
A nit: probably should `assertContract()` _before_ `contract.greet()`, in order to get a more helpful failure message in the case where `assertContract()` would fail.

Reply:
Right makes sense